### ISSUE TITLE
Turn off tools menu for viewing of model

### ISSUE THREAD

**Issue by softwarelifecycle** [potree/potree#1174]
I am using PoTree along with OpenDroneMap to display 3d photogrammetry models. I want to be able to hide the tools menu on the left and just allow users to view the model... because if they can get at the tools and add measurements, another user looking at the same model will see those annotations. That gets messy.. I mainly just want them to view the model..

Also, is there a way to have the textured model be displayed by default so it doesn't have to be manually turned on?

Thanks!
Harold

**Comment by HeadLessHUN** [comment]
Did opendronemap implement a feature to show measurements? Because by default potre does not have a server feature to store/share measurements, or any states of the tools. So unless opendronemap team implemented it. You don't have to do anything with that. As for mesh model loading. I think you cane do that but you will need to do something inside opendronemap because this feature is implemented there.

**Comment by softwarelifecycle** [comment]
Yes Opendronemap has a tools section for calculating distance, height, areas etc using squares, circles and polygons...  ok, thanks, I'll dig into the code..

**Comment by HeadLessHUN** [comment]
Potree has these tools i meant that they will disappear as soon as you refresh your page. Unless opendronemap saves these to a database. That saving part requires a server side script/database.
